test/client-library-secrets.py

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/python
+
+# Python is the common scripting language between the docker images used to
+# test the various Taskcluster client libraries.  This fetches the
+# `testing/client-libraries` secret and writes out shell-compatible export
+# statements for each value it contains.
```

Comment:
Let's highlight that this secret contains _taskcluster credentials_ (TASKCLUSTER_ACCESS_TOKEN / TASKCLUSTER_CLIENT_ID / TASKCLUSTER_ROOT_URL ?) required for the integration tests which connect to a real taskcluster deployment rather than a mock/fake.